### PR TITLE
add newly defined workflow variables models

### PR DESCRIPTION
**Pull request by tao-xinxiu**
Present newly defined workflow variable models:
- PA:NOT_EMPTY
- PA:HIDDEN
- PA:CREDENTIAL

### CODE REVIEW

File: src/docs/user/references/Variables.adoc [hunk 1]
```diff
@@ -132,6 +132,9 @@ Note that the revision sub-pattern is a hash code number represented by 13 digit
 Examples: PA:CATALOG_OBJECT will accept "bucket-example/object-example/1539310165443" and "bucket-example/object-example" but not "bucket-example/object-example/153931016" (invalid revision number) and "bucket-example/" (missing object name).
 * *PA:JSON* : variable syntax must be a valid JSON expression as defined in http://www.json.org/[JSON doc]. +
 Examples: PA:JSON will accept {"name": "John", "city":"New York"} and empty values like {} or [{},{}], but not ["test" : 123] (Unexpected character ':') and {test : 123} (Unexpected character 't').
+* *PA:NOT_EMPTY* : variable must be provided with a non-empty value.
+* *PA:HIDDEN* : variable which allows the user to securely enter its value (i.e., each character is shown as an asterisk, so that it cannot be read.) during submit the workflow.
```
Comment (mklkun) [inline src/docs/user/references/Variables.adoc]: during submit the workflow. =?=> during submitting the workflow. (I am not sure)

Comment (tao-xinxiu) [reply src/docs/user/references/Variables.adoc]: Thanks, I just found that "while" should be used before verb, I'm correcting it.

Comment (tao-xinxiu) [reply src/docs/user/references/Variables.adoc]: Thanks, I just found "while" should be used before verb. I'm correcting it.

Comment (mklkun) [reply src/docs/user/references/Variables.adoc]: You're welcome! Good.

**Pull request merged**